numba/tests/test_dyn_array.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import print_function, absolute_import, division
+
+import sys
+import numpy as np
+import threading
+
+from numba import unittest_support as unittest
+from numba import njit
+from numba import utils
+
+
+
+nrtjit = njit(nrt=True)
```

Comment:
I don't think it makes sense to expose "nrt" to the user. Numba should always allow memory allocation from CPU funcs, IMO.